content/departments/engineering/teams/core-services/managed-services/telemetry-gateway.md

```diff
@@ -0,0 +1,80 @@
+
+## Local development
+
+For local development, please refer to its [How to set up Telemetry Gateway locally](https://docs.sourcegraph.com/dev/how-to/telemetry_gateway).
+
+## Operations
+
+> [!NOTE]
+> To get access to most resources, you’ll need to [request infrastructure access](#infrastructure-access).
+
+Here is a list of useful quick links:
+
+- Prod instance: `telemetry-gateway.sourcegraph.com` - currently only accepts real Sourcegraph licenses.
+  - [Terraform Cloud workspaces](https://app.terraform.io/app/sourcegraph/workspaces?project=prj-9XNnACvkeM1VWteC)
+  - [Cloud Run service (metrics overview)](https://console.cloud.google.com/run/detail/us-central1/telemetry-gateway/metrics?project=telemetry-gateway-prod-acae)
+  - [Service logs](https://cloudlogging.app.goo.gl/kficDmGcZdMJHPQL9)
+  - [GCP alerts](https://console.cloud.google.com/monitoring/alerting?project=telemetry-gateway-prod-acae)
+  - [GCP errors](https://console.cloud.google.com/errors?project=telemetry-gateway-prod-acae)
+- Dev instance: `telemetry-gateway.sgdev.org` - currently only accepts `dev-private` licenses.
+  - [Terraform Cloud workspaces](https://app.terraform.io/app/sourcegraph/workspaces?project=prj-nxL7Ti7x8xp6oZTU)
+  - [Cloud Run service (metrics overview)](https://console.cloud.google.com/run/detail/us-central1/telemetry-gateway/metrics?project=telemetry-gateway-dev-0050)
+  - [Service logs](https://cloudlogging.app.goo.gl/4oVGWGz1FQKVt5vm9)
+  - [GCP alerts](https://console.cloud.google.com/monitoring/alerting?project=telemetry-gateway-dev-0050)
+  - [GCP errors](https://console.cloud.google.com/errors?project=telemetry-gateway-dev-0050)
+
+### Infrastructure access
+
+The following Entitle requests are needed to get access to Telemetry Gateway service infrastructure:
+
+- [GCP project - Prod Editor](https://app.entitle.io/request?targetType=resource&duration=21600&justification=Justification%20here&integrationId=134476cb-0bd6-4c6d-a89f-e1550988bdd7&resourceId=271c1799-6172-4099-8fe1-b186ac05aa06&roleId=da83e573-1cae-4e83-a132-3d2c6065ecbb&grantMethodId=da83e573-1cae-4e83-a132-3d2c6065ecbb)
+
+All engineers should have access to the dev project by default.
+
+### Deployment
+
+The Telemetry Gateway service infrastructure is defined in [`sourcegraph/managed-services/services/telemetry-gateway`](https://github.com/sourcegraph/managed-services/tree/main/services/telemetry-gateway) utilizing [Managed Services Platform](./platform.md).
+
+#### Modify deployment manifest
+
+> [!WARNING]
+> Due to the early-stage shape of Managed Services Platforms, we have yet to roll out standardized playbook. Please reach out to #team-core-services for modifying the deployment manifest. Instructions in this section are generally assumed with an upfront setup.
+
+To modify the deployment manifest:
+
+1. Update `service.yaml` file
+1. In the repository root, run `sg msp generate services/telemetry-gateway/service.yaml prod`
```

Comment:
This could be updated with the auto root-finding feature 😁 
```suggestion
1. Anywhere in the repository, run `sg msp generate telemetry-gateway prod`
```